src/components/DetailsPanel/InterpretationsUnit/Interpretation.js

```diff
@@ -0,0 +1,113 @@
+import classes from './styles/Interpretation.module.css'
+
+export const Interpretation = ({
+    interpretation,
+    currentUser,
+    onClick,
+    onLikeToggle,
+}) => {
+    const onComplete = res => {
+        if (res.status === 'OK') {
+            onLikeToggle()
+        }
+    }
+    const resource = `interpretations/${interpretation.id}/like`
+    const likeMutationRef = useRef({ resource, type: 'create' })
+    const unlikeMutationRef = useRef({ resource, type: 'delete' })
+    const [like] = useDataMutation(likeMutationRef.current, { onComplete })
+    const [unlike] = useDataMutation(unlikeMutationRef.current, { onComplete })
+    const [isLikedByCurrentUser, setIsLikedByCurrentUser] = useState(false)
+    const onLikeClick = isLiked => e => {
+        e.stopPropagation()
+        isLiked ? unlike() : like()
+    }
+
+    useEffect(() => {
+        setIsLikedByCurrentUser(
+            interpretation.likedBy.some(
+                likedBy => likedBy.id === currentUser.id
+            )
+        )
+    }, [currentUser, interpretation])
+
+    return (
+        <li
+            className={classes.interpretationSection}
+            onClick={() => onClick(interpretation.id)}
+        >
+            <div className={classes.interpretationHeader}>
+                <UserAvatar name={interpretation.user.displayName} extrasmall />
+                {interpretation.user.displayName}
+                <time dateTime={interpretation.created}>
+                    {moment(interpretation.created).format('DD/MM/YY hh:mm')}
+                </time>
+            </div>
+            <div className={classes.interpretationContent}>
+                <RichTextParser>{interpretation.text}</RichTextParser>
+            </div>
+            <div className={classes.interpretationStats}>
+                <Tooltip closeDelay={200} content={i18n.t('Like')}>
+                    <span
+                        onClick={onLikeClick(isLikedByCurrentUser)}
+                        className={cx({
+                            [classes.isLiked]: isLikedByCurrentUser,
+                        })}
+                    >
+                        {interpretation.likes ? interpretation.likes : null}{' '}
+                        <IconThumbUp16
+                            color={
+                                isLikedByCurrentUser
+                                    ? colors.teal500
+                                    : colors.grey700
+                            }
+                        />
+                    </span>
+                </Tooltip>
+                <Tooltip closeDelay={200} content={i18n.t('Reply')}>
+                    <span onClick={() => onClick(interpretation.id)}>
+                        {interpretation.comments.length
+                            ? interpretation.comments.length
+                            : null}
+                        <IconReply16 color={colors.grey700} />
+                    </span>
```

Comment:
Semantically a `<button>` tag is probably a better fit here. With our supported browsers list we can use the CSS [`all` property](https://caniuse.com/css-all) to unset the browser styles (which effectively should make this look like a span?):
```css
button {
    all: unset;
    /* More styles */
}